src/Drivers/ObjectDriver.php

```diff
@@ -12,6 +12,11 @@
 
 class ObjectDriver implements Driver
 {
+    public function __construct(
```

Comment:
Instead of this single parameter, let's allow passing an array with options. 

The default options should be 

```
  'yaml_inline' => 2,
  'yaml_indent' => 4,
  'yaml_flags' => Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK,
```

Any options passed should be merged with the defaults.

Reply:
> If you are fine to have a test for only the inline var, I can do it in my PR.

That's ok!

Reply:
Cool! Requested changes are done!